docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings.md

```diff
@@ -54,20 +62,27 @@ For instance, the sample JSON snippet below is a tree with nested resources to r
         },
    ...
 ```
+ Within the **Coiote DM - Azure IoT Central integration**, readable and writable resources are interpreted as Properties. While in Azure IoT Hub the data structure is nested, in Azure IoT Central it is represented in one line where objects, object instances, and resources are separated by `n`. For example: `n1n1n1`.
+
+In Azure IoT Central, Properties as well as other device capabilities (Telemetry and Command, read more on them below) are defined in a model which is part of the [device template](https://docs.microsoft.com/en-us/azure/iot-central/core/concepts-device-templates). The device template defines the characteristics of the device that is connected to your IoT Central application.
 
 ### READ - Communication flow
 
-Data model resources that are read-only, such as `Manufacturer` (with ID **3/0/0**) will be mapped into the Device twin as a reported property.
+Data model resources that are read-only, such as `Manufacturer` (with ID **3/0/0**) will be mapped into the Device twin as a reported property both in Azure IoT Hub and Azure IoT Central.
```

Comment:
to jest napisane both in Azure Iot Huba and .. Central, a już poniżej jak mowa o device-to-cloud, to jest tylko Hub

Reply:
dzięki za uwagę, popawię to